data_steward/validation/participants/identity_match.py

```diff
@@ -1,379 +1,336 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
 """
 A module to perform initial participant identity matching based on PII data.
 
 Compares site PII data to values from the RDR, looking to identify discrepancies.
 """
+# Python imports
+from datetime import datetime
+import json
```

Comment:
This can be removed if we use the resources.

Reply:
I still need the datetime to provide the date for the new match dataset name.  I have removed the json import.